source/SH3/arc/subarc.cpp

```diff
@@ -28,6 +28,8 @@ namespace {
         std::uint32_t numFiles;          /**< Number of files located in this sub .arc */
         std::uint32_t dataPointer;       /**< Pointer to the beginning of the data section */
         std::uint32_t unused;            /**< Unused DWORD */
+
+        static constexpr std::uint32_t expectedMagic = 0x20030507; /**< Expected value of @ref magic. */
```

Comment:
Should this be `ALL_CAPS` for a static constant??

Reply:
It seems we're a little bit inconsistent. See `arcFileNotFound` or our `sh3_gl::vao_target_array`s for instance. We should perhaps try to define a `.clang-format` and run a pass on our code.

Reply:
Hmm I see, I'm surprised that one slipped through as well (maybe when we refactored it).

Can we define our own formatting style with .clang-format (so that the codebase still follows the Allman style)?